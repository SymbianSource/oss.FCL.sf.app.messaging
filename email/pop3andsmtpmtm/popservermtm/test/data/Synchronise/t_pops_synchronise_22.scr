+OK demon POP3 server ready (demon.co.uk)
+OK 
+OK 
+OK 6 7867
+OK 6 messages
1 1316
2 1401
3 1278
4 1235
5 1319
6 1318
.
+OK 6 messages
1 yashar.997723757:20:02036:353
2 yashar.997726929:20:19209:1071
3 yashar.997722875:21:28519:716
4 yashar.997725402:20:09569:43
5 yashar.997786980:20:10598:249
6 yashar.997785992:10:04536:10
.
-ERR Unknown Command "CAPA"
+OK 
Return-Path: <Andrew.Fawcett@teleca.com>
Received: from punt-1.mail.demon.net by mailstore for freddy@yashar.demon.co.uk
          id 997785992:10:04536:10; Tue, 14 Aug 2001 10:46:32 GMT
Received: from mail.teleca.com ([195.166.58.131]) by punt-1.mail.demon.net
           id aa1117959; 14 Aug 2001 10:46 GMT
Received: from teleca126.teleca.com (unverified) by teleca161.teleca.com
 (Content Technologies SMTPRS 4.1.5) with ESMTP id <Tac100102555d4cc331@teleca161.teleca.com> for <freddy
@yashar.demon.co.uk>;
 Tue, 14 Aug 2001 11:45:31 +0100
Received: by teleca126 with Internet Mail Service (5.5.2650.21)
        id <Q44VG0CR>; Tue, 14 Aug 2001 11:46:13 +0100
Message-ID: <t_pops_synchronise_message008@JANUS.teleca.com>
From: Andrew Fawcett <Andrew.Fawcett@teleca.com>
To: 'freddy' <freddy@yashar.demon.co.uk>
Subject: Message 8
Date: Tue, 14 Aug 2001 11:38:26 +0100
Return-Receipt-To: Andrew Fawcett <Andrew.Fawcett@teleca.com>
MIME-Version: 1.0
X-Mailer: Internet Mail Service (5.5.2650.21)
Content-Type: text/plain;
        charset="iso-8859-1"

Eight Eight Eight Eight Eight Eight Eight Eight Eight Eight  
.
+OK 
Return-Path: <Andrew.Fawcett@teleca.com>
Received: from punt-2.mail.demon.net by mailstore for freddy@yashar.demon.co.uk
          id 997786980:20:10598:249; Tue, 14 Aug 2001 11:03:00 GMT
Received: from mail.teleca.com ([195.166.58.131]) by punt-2.mail.demon.net
           id aa2002162; 14 Aug 2001 10:44 GMT
Received: from teleca126.teleca.com (unverified) by teleca161.teleca.com
 (Content Technologies SMTPRS 4.1.5) with ESMTP id <Tac100102555d4c3c4d@teleca161.teleca.com> for <freddy
@yashar.demon.co.uk>;
 Tue, 14 Aug 2001 11:44:57 +0100
Received: by teleca126 with Internet Mail Service (5.5.2650.21)
        id <Q44VG0C3>; Tue, 14 Aug 2001 11:45:32 +0100
Message-ID: <t_pops_synchronise_message007@JANUS.teleca.com>
From: Andrew Fawcett <Andrew.Fawcett@teleca.com>
To: 'freddy' <freddy@yashar.demon.co.uk>
Subject: Message 7
Date: Tue, 14 Aug 2001 11:37:44 +0100
Return-Receipt-To: Andrew Fawcett <Andrew.Fawcett@teleca.com>
MIME-Version: 1.0
X-Mailer: Internet Mail Service (5.5.2650.21)
Content-Type: text/plain;
        charset="iso-8859-1"

Seven Seven Seven Seven Seven Seven Seven Seven Seven Seven  
.
** Read cancelled
+OK 
