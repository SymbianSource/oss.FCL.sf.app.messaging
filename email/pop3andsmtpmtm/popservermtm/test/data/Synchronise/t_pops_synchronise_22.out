STAT
LIST
UIDL
CAPA
TOP 6 1
TOP 5 1
QUIT
